azuredisk-csi.yaml

```diff
@@ -0,0 +1,71 @@
+package:
+  name: azuredisk-csi
+  version: 1.31.0
+  epoch: 0
+  description: Azure Disk CSI Driver
+  copyright:
+    - license: Apache-2.0
+
+environment:
```

Comment:
thinking if we have a concrete reason to use CGO_ENABLED=0 here, generally we leave it to the build pipeline.

Reply:
it was an old copy pasta thing that came up here, will remove it, thanks for the review